ui/apps/platform/src/test-utils/ComponentProviders.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { ApolloProvider } from '@apollo/client';
+import { createBrowserHistory } from 'history';
+
+import configureApolloClient from 'configureApolloClient';
+
+// Provides the base application providers for testing
+export default function ComponentTestProviders({ children }: { children: React.ReactNode }) {
+    const history = createBrowserHistory();
+
+    return (
+        <Router history={history}>
+            <ApolloProvider client={configureApolloClient()}>{children}</ApolloProvider>
+        </Router>
+    );
+}
```

Comment:
A base wrapper of context providers for components that reference context consumers. Note that we are using the regular `ApolloProvider` here instead of the `MockProvider`. This is to keep consistency with the REST style endpoints and e2e tests and allowing mocking of GraphQL calls with `cy.intercept`.

We will likely want to add a context for Redux here in the future.